json-path/src/test/java/com/jayway/jsonpath/old/FilterTest.java

```diff
@@ -348,6 +350,12 @@ public boolean apply(PredicateContext ctx) {
                 }
                 return false;
             }
+
+			@Override
+			public void getRelationalExprValues(ArrayList<SimpleEntry<String, String>> valuesMap) {
+				// TODO Auto-generated method stub
+				
+			}
```

Comment:
Can you add a couple of tests for the filter operators listed in https://github.com/json-path/JsonPath ? make sure you do in, nin, subset-of, size, empty.  You can add a couple for relational operators don't have to cover all.

Reply:
Done.